### PR TITLE
zh-translation:/docs/reference/glossary/mixer-instance.md

### PR DESCRIPTION
**Pull request by Lovnx**
ref: https://github.com/servicemesher/istio-official-translation/issues/1370

### CODE REVIEW

**Comment by rootsongjc** [comment]
Please sign the CLA first. @Lovnx

**Comment by Lovnx** [comment]
@googlebot I signed it!

File: content/zh/docs/reference/glossary/mixer-instance.md [hunk 1]
```diff
@@ -2,6 +2,4 @@
 title: Mixer Instance
 ---
 
-An instance represents a chunk of Mixer data that is produced by inspecting a set of request [attributes](#attribute) and applying the operator-supplied configuration.
-Instances are delivered to individual [handlers](#mixer-handler), on their way to
-infrastructure backends.
+一个 Mixer Instance 表示通过检查一组请求[属性](#attribute) ，并结合使用者提供的配置，而生成的数据块。 Mixer Instance 在随请求到达各种基础后端设施的途中，会被传递给各个[处理程序](#mixer-handler)
```
Comment (rootsongjc) [inline content/zh/docs/reference/glossary/mixer-instance.md]: 开头的 `一个` 可以去掉。“而生成”前的逗号可以去掉。第二个 Mixer Instance 前多了一个空格。

**Pull request merged**